src/core/lib/layer-manager.js

```diff
@@ -202,6 +204,17 @@ export default class LayerManager {
   }
 
   updateLayers({newLayers}) {
+    // Filter out any null layers
+    newLayers = newLayers.filter(newLayer => newLayer !== null);
+
+    // Update context object if it is changed.
+    if (this._isContextDirty) {
+      for (const layer of newLayers) {
+        layer.context = this.context;
+      }
+      this._isContextDirty = false;
+    }
+
     // TODO - something is generating state updates that cause rerender of the same
     if (newLayers === this.lastRenderedLayers) {
```

Comment:
This will never be true since `filter` constructs a new array.

Reply:
Good catch, fixed.